client/tests/smokeTest.test.js

```diff
@@ -0,0 +1,122 @@
+const puppeteer = require('puppeteer');
+const path = require('path');
+const spawn = require('cross-spawn');
+const treeKill = require('tree-kill');
+
+// TODO: In a future iteration the server start and close functions should
+// handled in one place by Jest's setup and teardown scripts
+
+const startServer = async serverOrClient => {
+    return new Promise(resolve => {
+        const server = spawn('yarn', ['workspace', serverOrClient, 'dev'], {
+            cwd: path.resolve(__dirname, '../../'),
+            env: {
+                PATH: process.env.PATH,
+                PORT: 8033,
+                CLIENT_PORT: 3033,
+                AUTOMATION_SCHEDULER_PORT: 8833,
+                API_SERVER: 'http://localhost:8033',
+                APP_SERVER: 'http://localhost:3033',
+                AUTOMATION_SCHEDULER_URL: 'http://localhost:8833',
+                PGDATABASE: 'aria_at_report_test',
+                PGPORT: 5432,
+                ENVIRONMENT: 'test'
+            }
+        });
+
+        const killServer = async () => {
+            await new Promise((resolve, reject) => {
+                treeKill(server.pid, error => {
+                    if (error) return reject(error);
+                    resolve();
+                });
+            });
+        };
+
+        server.stdout.on('data', data => {
+            const output = data.toString();
+            console.info(output); // eslint-disable-line no-console
+
+            if (
+                (serverOrClient === 'server' &&
+                    output.includes('Listening on 8033')) ||
```

Comment:
I think you could make the static env values being referenced above into file scoped constants, so they could be reused in instances like this (`PORT` instead of "8033").

This could be beneficial in the instance that the variables do have to change, or if there's need for dynamic re-assignments in the future.